lib/service/cache/redis.js

```diff
@@ -93,6 +96,34 @@ class Redis extends Service {
     });
   }
 
+  /**
+   * Setup a ping interval to keep the connection alive
+   * Every 60 seconds a ping is sent to Redis
+   */
+  _setupKeepAlive() {
+    this.client.on('ready', async () => {
+      await this._ping();
+      this.pingIntervalID = setInterval(this._ping.bind(this), 60000);
```

Comment:
I personnaly don't like hardcoded value in code. You could create a new configuration field to:
- enable/disable the ping mechanism
- configure the ping delay

Reply:
This new configuration should be use later when IoRedis ping implementation will be ready

Reply:
And I think this should be disabled by default to avoid edge effects on other installation that do not require it

Reply:
I did add a config that enables the Ping Keep Alive and sets its delay